.gitignore

```diff
@@ -21,6 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```

Comment:
@sumana2001, should we keep this file in gitignore or not?

Reply:
It's ok to keep